assets/js/blocks/cart/cart-cross-sells-product-list/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { ProductResponseItem } from '@woocommerce/type-defs/product-response';
+
+/**
+ * Internal dependencies
+ */
+import CartCrossSellsProduct from './cart-cross-sells-product';
+
+const placeholderRows = [ ...Array( 3 ) ].map( ( _x, i ) => (
+	<CartCrossSellsProduct crossSellsProduct={ {} } key={ i } />
+) );
+
+interface CrossSellsProductListProps {
+	crossSellsProducts: ProductResponseItem[];
+	isLoading: boolean;
+	className?: string;
+	columns: number;
+}
+
+const CartCrossSellsProductList = ( {
+	crossSellsProducts,
```

Comment:
Would `products` be a better name? the `crossSells` doesn't seem to add much value.

Reply:
See the comment I added to https://github.com/woocommerce/woocommerce-blocks/pull/6645/files/aed95be381fedeb379eabc460a78e591b8106a61#r956142844.

Reply:
I don't think that comment applies here, I was referring to the prop `crossSellsProducts` - `products` seems like it would be fine?

Reply:
I see now what you meant and just changed it! Thanks for pointing this out, @opr.